python/ray/experimental/channel/gpu_communicator.py

```diff
@@ -16,7 +16,7 @@
 @DeveloperAPI
 class GPUCommunicator(ABC):
     """
-    Communicator for a group of aDAG actors on Nvidia GPU.
+    Communicator for a group of aDAG actors on Nvidia GPU or other XPUs.
```

Comment:
We should probably change the class name to a more general one if this is to support other XPUs. This is not yet used externally so backward compatibility is not an issue.

Reply:
I agree. Next step I prefer to change it to `AcceleratorCommunicator` or just `Communicator` for all. Currently, this `GPUCommunicator` is also called from some top level so I just keep it now.